test/inductor/test_torchinductor_dynamic_shapes.py

```diff
@@ -47,10 +47,6 @@
     #
     # These are from switching to specialize_int=False
     #
-    "test_div5_dynamic_shapes": (
-        "cpu",
-        "cuda",
-    ),  # The values for attribute 'dtype' do not match
```

Comment:
Fixes this. Only `floor` and `trunc` are required for this one, but I changed the other two rounding funcs to be consistent.

Reply:
The cause of the error is the out dtype. It was previously coming from the first input arg (see `dtype` in `make_pointwise`), which generates the `empty_strided` line via `make_buffer_allocation`. Before vs after:

```
extern "C" void kernel(const long* in_ptr0,
                       [-float*-]{+long*+} out_ptr0,
                       const long ks0,
                       const long ks1)
{
...
    buf0 = empty_strided((s0, s0), (s0, 1), device='cpu', [-dtype=torch.float32)-]{+dtype=torch.int64)+}

```